Plugins/Public/base_plugin/Main.cpp

```diff
@@ -84,6 +84,7 @@ map<wstring, RECIPE> moduleNameRecipeMap;
 map<uint, RECIPE> moduleNumberRecipeMap;
 map<wstring, map<uint, RECIPE>> craftListNumberModuleMap;
 set<wstring> buildingCraftLists;
+map<uint, RECIPE> blueprintRecipeMap;
```

Comment:
Use unordered_map for optimization purposes (other maps above are already updated on my repo)